### ISSUE TITLE
WD SN750  3rd party NVMe controller. Loss of MMIO space.

### ISSUE THREAD

**Issue by lukasa1993** [acidanthera/bugtracker#2259]
Got brand new SN750 as recommended but still get kernel crash i have 3 m.2 now old SN550 some Gigabit and some netec but all of them get kernel crash 

```
panic(cpu 0 caller 0xffffff8016c1e674): nvme: "3rd party NVMe controller. Loss of MMIO space. Write. fBuiltIn=1 MODEL=WDBRPG5000ANC-WRSN FW=111130WD CSTS=0xffffffff US[1]=0x0 US[0]=0xb VID=0x15b7 DID=0x5006 CRITICAL_WARNING=0x0.\n" @IONVMeController.cpp:6147
```

**Comment by lukasa1993** [comment]
all is clean, and it works fine on low load but on high load it crashes

**Comment by lukasa1993** [comment]
> @lukasa1993 
> 
> I have 2 SN750 (device-ids 0x5002,0x15b7 and 0x5006,0x15b7) and both are working fine. Ventura + OpenCore 0.9.1 + NVMeFix.kext.
> 
> 
> 
> I have read this panic somewhere in the past, even on real Macs. 
> 
> 
> 
> Try this: turn off the computer, clean carefully the PCIe connectors of the drives (especially the new) and plug them back in ensuring they are well connected.
> 
> 
> 
> See also:
> 
> [Issue 1961](https://github.com/acidanthera/bugtracker/issues/1961)
> 
> 

what motherboard are you using?

**Comment by perez987** [comment]
@lukasa1993 
Z390 Aorus Elite + i9-9900k + 6600 XT.

**Comment by lukasa1993** [comment]
> @lukasa1993 
> 
> Z390 Aorus Elite + i9-9900k + 6600 XT.

and have you tried them with very high load ?

**Comment by lukasa1993** [comment]
> @lukasa1993
> Amorphous DiskMark working for 30 minutes, reading and writing on the macOS NVMe disk. No issues so far.
> I think this can be for a support forum.

i have high cpu + high ssd load i forgot to mention

**Comment by perez987** [comment]
@lukasa1993
Not me. No high CPU and disks load.

**Comment by lukasa1993** [comment]
> @lukasa1993
> 
> Not me. No high CPU and disks load.

i mean when it panics i have high cpu and ssd load in low loads it works

**Comment by perez987** [comment]
@lukasa1993 
Tell me how do you do in your system to force CPU and disk so I can try in the same way.

**Comment by lukasa1993** [comment]
not sure how to make it synthetic but when lots of stuff is going on building code watching video downloading something it usually panics

**Comment by perez987** [comment]
@lukasa1993 
Simultaneously building OpenCore + creating a big file by dd in Terminal + playing a high quality video. No crash, all seems to be ok.

**Comment by lukasa1993** [comment]
can it be motherboard thing ?

**Comment by perez987** [comment]
Not sure. But it's better to go to a support forum where you'll get help.

**Comment by lukasa1993** [comment]
> Not sure. But it's better to go to a support forum where you'll get help.

thanks for everything where i go can you link ?

**Comment by perez987** [comment]
https://www.insanelymac.com/forum/

https://www.tonymacx86.com/

You can close the issue.

**Comment by lukasa1993** [comment]
oh i have there no luck so far thank you

**Comment by perez987** [comment]
@lukasa1993 
But don’t leave the issue open, close it.

**Comment by extric99** [comment]
I also have this issue and it is not related to clean connectors 
When I put the NVME in my main PC with a Gibabyte motherboard I get the error but when I move the NMVE to my NUCintosh all is well. I have had the issue with a Samsung 970 and three different models of WD (SN550,SN570 and SN750). Also I tried putting the nvme in a USB enclosure and the issue still occurs which puzzles me even more.

I also switched from Gigabyte Z390 Gaming  X to a Gigabyte Z390m  Gaming and the issue occurs on both systems. 
Windows11 on the same system is rock solid so I am puzzled at what makes MacOS freak out. Multiple people have this issue

https://www.tonymacx86.com/threads/nvme-kernel-panic-out-of-ideas.316473/page-3
